doc/using.rst

```diff
@@ -1,128 +1,111 @@
-  notebooks or scripts.
-* Configuration files (one or more text files) that specify the requirements
-  for building your project's code. For a complete list of supported files,
-  see :ref:`config-files`. Configuration files may be placed in the root of
-  your repository or in a ``binder/`` folder in the repository's root
-  (i.e. ``myproject/binder/``). If a ``binder/`` folder is used, Binder will
-  only read configuration files from that location and will ignore those in
-  the repository's root.
-
-.. tip::
+For a more complete list of sample repositories for use with Binder, see the
+`Sample Binder Repositories <sample_repos.html>`_ page.
 
-   For a list of sample repositories for use with Binder, see the
-   `Sample Binder Repositories <sample_repos.html>`_ page.
+Choose from many open source languages
+======================================
 
-Launching your Binder
-=====================
+Binder runs on JupyterHub, a language-agnostic tool for serving data
+analytics sessions in the cloud. This means that you can build Binders
+using many popular open source languages.
 
-Once you have prepared your repository per the instructions above,
-it is time to build and launch your binder-ready repo. Navigate to ``mybinder.org``
-and insert the URL for your git repository. Press ``Launch`` to automatically create your Binder.
-The Binder service will automatically send you to a live Jupyter session
-connected to this repository.
+For example, here's a short list of Binder repositories that are built
+with several popular languages:
 
-If a previous version of the repository has already been built, Binder will
-only build a new one if the git hashes don't match. When Binder
-*doesn't* need to build a repository, the process of connecting to the live
-computational environment is much faster.
+* Python
+    * `A simple pip environment <https://github.com/binder-examples/requirements>`_.
+    * `An Anaconda Python environment <https://github.com/binder-examples/conda>`_.
+* R
+    * `Using R, RStudio, and Shiny <https://github.com/binder-examples/r>`_.
+* Julia
+    * `A simple Julia repository <https://github.com/binder-examples/julia-python>`_.
 
-Use cases
-=========
+You can also control the versions used for each of these languages. For
+more information about this, see the :ref:`config-files` page.
 
-Below are some example use-cases and their respective repository
-configuration.
+Choose from multiple user interfaces
+====================================
 
-For a full guide on how to configure your repository for
-Binder, see the `repo2docker configuration guide
-<http://repo2docker.readthedocs.io/en/latest/usage.html#preparing-your-repository>`_.
-For a list of sample repositories for use with Binder, see the
-`Sample Binder Repositories <sample_repos.html>`_ page.
+Binder is also agnostic to the specific user interface that you'd like
+to share with your Binder. BinderHub will make some assumptions about the
```

Comment:
This is another challenging one - "Binder" is being used as two different things in this sentence, first as a service, then as a "binderhub + repo". Can we exchange the first "Binder" with "BinderHub" ?